bloom-housing/bloom
# feat: creating users as admin

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2802)

**Relies on work of:** https://github.com/bloom-housing/bloom/pull/2848


## Description
This updates how creating a user works as an admin user

## How Can This Be Tested/Reviewed?
Log in as an admin user
go to the user grid:
Create a new admin user, notice the jurisdiction/listing dropdowns no longer show

create a jurisdictional admin, notice only the jurisdictional dropdown shows

create a partner, notice the jurisdiction checkboxes and once a jurisdiction is selected, the listings checkboxes shows

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
